radis/test/lbl/test_base.py

```diff
@@ -400,12 +402,161 @@ def test_optically_thick_limit_2iso(verbose=True, plot=True, *args, **kwargs):
         radis.DEBUG_MODE = DEBUG_MODE
 
 
+def test_get_waverange(*args, **kwargs):
```

Comment:
Great test suite!

I would also report here the comments of your [lab notebook](https://github.com/pkj-m/radis-lab/blob/master/lbl/Testing%20the%20new%20get_waverange%20method.ipynb), adding the "expected behavior". That will greatly improve the future readability of your test suite, if someone ever needs to update it:

Ex:
```
# wunit is none 
# ... wmin/wmax is none
# ...... wavenumber is passed  > should return the same wavenumbers

# ...... wavelength is passed > should convert and return wavenumbers

# ...... both wavelengths & wavenumbers are passed > should raise an Error